### ISSUE TITLE
Support 1.11.2 in CB-Reflect

### ISSUE THREAD

**Issue by mibby** [NoCheatPlus/Issues#345]
PaperSpigot dev 991 (MC/Spigot 1.11.2)
NoCheatPlus dev 1046
ProtocolLib dev 345

❤️ 

```
[18:49:34] [Server thread/INFO]: [NoCheatPlus] Enabling NoCheatPlus v3.15.0-SNAPSHOT-sMD5NET-b1046
[18:49:34] [Server thread/WARN]: [NoCheatPlus] The Minecraft version seems to be more recent than the one Compat-CB-Reflect has been built with - this might work, but there could be incompatibilities.
[18:49:34] [Server thread/INFO]: [NoCheatPlus] McAccess set to: 1.4.5-1.11|? / CB-Reflect
[18:49:34] [Server thread/INFO]: [NoCheatPlus] Added block-info for Minecraft 1.5 blocks.
[18:49:34] [Server thread/INFO]: [NoCheatPlus] Added block-info for Minecraft 1.6.1 blocks.
[18:49:34] [Server thread/INFO]: [NoCheatPlus] Added block-info for Minecraft 1.7.2 blocks.
[18:49:34] [Server thread/INFO]: [NoCheatPlus] Added block-info for Minecraft 1.8 blocks.
[18:49:34] [Server thread/INFO]: [NoCheatPlus] Added block-info for Minecraft 1.9 blocks.
[18:49:34] [Server thread/INFO]: [NoCheatPlus] Added block-info for Minecraft 1.10 blocks.
[18:49:34] [Server thread/INFO]: [NoCheatPlus] Added block-info for Minecraft 1.11 blocks.
[18:49:34] [Server thread/INFO]: [NoCheatPlus] Inventory checks: FastConsume is available, disabled InstantEat.
[18:49:34] [Server thread/INFO]: [NoCheatPlus] Adding packet level hooks for ProtocolLib (MC 1.11.1)...
[18:49:34] [Server thread/INFO]: [NoCheatPlus] Available (and activated) packet level hooks: UseEntityAdapter | MovingFlying | OutgoingPosition | KeepAliveAdapter | SoundDistance
[18:49:34] [Server thread/INFO]: [NoCheatPlus] Force disable FastHeal on Minecraft 1.9 and later.
[18:49:34] [Server thread/INFO]: [NoCheatPlus] Version 3.15.0-SNAPSHOT-sMD5NET-b1046 is enabled.
```

**Comment by asofold** [comment]
Looks like the shapes are reported correctly, but i'll delay adapting the support-range check until i've had a look at the code.

**Comment by FirstReplay** [comment]
On version 1046 and Paper 990, I am seeing the following when booting the server:  

"[NoCheatPlus] The Minecraft version seems to be more recent than the one Compat-CB-Reflect has been built with - this might work, but there could be incompatibilities"

**Comment by asofold** [comment]
Yes i haven't updated it to officially allow the new version. It appears to work, but i won't alter it until i have really checked the code.

The reflection module does check method/field specificiations including return types, so it won't call random methods. just so.

**Comment by asofold** [comment]
Assume fixed, will re-check with 1.12.
